src/types.c

```diff
@@ -80,7 +80,7 @@ struct type
 	   tp_short, tp_unsigned_short,
 	   tp_int, tp_unsigned_int,
 	   tp_long, tp_unsigned_long,
-	   tp_long_long, tp_unsigned_long_long,
+	   tp_long_long, tp_unsigned_long_long, tp_Bool,
 	   /* Used as the rep type of enums whose constants are derived
```

Comment:
- from C11: The rank of _Bool shall be less than the rank of all other standard integer types.
-> following the comment at the start of this enum, tp_Bool must be before tp_char
[And that also means there's a missing test case...]

Reply:
And some changes will likely follow elsewhere btw, I'm sure the code assumes tp_char is the lowest rank (e.g., see the assert in common_primitive_type).